frontend/src/pages/ActivityReport/Pages/activitySummary.js

```diff
@@ -233,6 +237,38 @@ const ActivitySummary = ({
     </div>
   );
 
+  const validateCitations = () => {
+    const allGoals = [selectedGoals, goalForEditing].flat().filter((g) => g !== null);
+    // If we have a monitoring goal.
+    const selectedMonitoringGoal = allGoals.find((goal) => goal.standard === 'Monitoring');
+    if (selectedMonitoringGoal) {
+      // Get all the citations in a single array from all the goal objectives.
+      const allCitations = selectedMonitoringGoal.objectives
+        .map((objective) => objective.citations)
+        .flat()
+        .filter((citation) => citation !== null);
```

Comment:
Likewise here, if there's a citation, could be it be null for some reason? If so, I wonder why

Reply:
I wonder if this was a bi product of not having the initial check above. If anything it prevents a possible error.